app/services/user_service/user_musics_destroyer.rb

```diff
@@ -0,0 +1,41 @@
+module UserService
+    class UserMusicsDestroyer < ApplicationService
+        def initialize(args)
+            @request = args[:request]
+            @music_ids = args[:music_ids]
+        end
+
+        def call
+            auth_result = AuthService::Authorizer.call(request: @request)
+            return auth_result if auth_result.is_a?(Hash)
+            @user_id = auth_result
+            
+            results = []
+            @music_ids.each do |music_id|
+                result = { destroyed_music_id: music_id }
+                begin
+                    user = User.find(@user_id)
+                rescue
+                    return {success: false, message: "User Not Found"}
```

Comment:
Service 내에서 exception handling을 하기보다는, 서비스를 호출하는 외부에서 하는 것이 더 좋을 것 같습니다. 

```suggestion
user = User.find(@user_id)
raise ActiveRecord::RecordNotFound unless user
```